File: relationship-manager/reminder-service/src/main/java/com/bank/rm/reminder/ReminderServiceApplication.java
package com.bank.rm.reminder;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;
@SpringBootApplication(scanBasePackages = {"com.bank.rm.reminder", "com.bank.rm.common"})
@EnableScheduling
public class ReminderServiceApplication {
    public static void main(String[] args) { SpringApplication.run(ReminderServiceApplication.class, args); }
}

File: relationship-manager/reminder-service/src/main/kotlin/com/bank/rm/reminder/ReminderServiceApplication.kt
package com.bank.rm.reminder

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication(scanBasePackages = ["com.bank.rm.reminder", "com.bank.rm.common"])
@EnableScheduling
class ReminderServiceApplication

fun main(args: Array<String>) {
    runApplication<ReminderServiceApplication>(*args)
}
